lib/metatags/Metatags.js

```diff
@@ -1,46 +1,34 @@
 "use strict";
 
+var CustomTags = require("./CustomTags");
+
 function createOpenGraphTag(property, content) {
+    console.warn("[DEPRECATED] Creating opengraph tags through Metatags has been deprecated. Please use new OpenGraphTags()");
     return '<meta property="' + property + '" content="' + content + '" data-ephemeral="true">';
 }
 
-function createNormalTag(name, content) {
-    return '<meta name="' + name + '" content="' + content + '" data-ephemeral="true">';
-}
-
 function createLinkTag(rel, href) {
+    console.warn("[DEPRECATED] Creating canonical tags through Metatags has been deprecated. Please use new LinkTags()");
```

Comment:
Was planning on doing that cool logger in another request. will put in a temp fix here